### PR TITLE
Instructor: edit session: show a spinner while saving session #8792

### LINKED ISSUE #8792: Instructor: edit session: show a spinner while saving session
v6.5.1

Steps:
1. Login as instructor
1. Edit session details
1. Save

image

Current: The `save changes` button stays active and can be clicked multiple times while the saving is in progress
Suggested: Disable the button while the saving is in progress and show a spinner e.g., `saving...`. I believe we do a similar thing in other places.

### PR DESCRIPTION
**Pull request by pierreSquared**
Fixes #8792 

I noticed there was no activity on this issue for over two weeks, so I went ahead and fixed it.

We already have a solution for a very similar issue. I used it.

### CODE REVIEW

**Comment by pierreSquared** [comment]
Ready for review.

I used the solution from #5858, the effect is the same as there.

File: src/main/webapp/dev/js/main/instructorFeedbackEdit.js [hunk 1]
```diff
@@ -311,6 +313,7 @@ function bindFeedbackSessionEditFormSubmission() {
                 }
 
                 if (!hasError) {
+                    removeLoadingIndicator($('#button_submit'), 'Save Changes');
```
Comment (whipermr5) [inline src/main/webapp/dev/js/main/instructorFeedbackEdit.js]: The spinner should be removed even when there's an error returned

Comment (pierreSquared) [reply src/main/webapp/dev/js/main/instructorFeedbackEdit.js]: You are right, of course. I fixed that.

**Comment by pierreSquared** [comment]
Ready for review.

The indicator is removed even in case of errors.

**Comment by pierreSquared** [comment]
Could you take a look a it. Thank you

**Pull request merged**